site/releases/2024-feb-14/highlights.qmd

```diff
@@ -0,0 +1,236 @@
+#### Update events to show users who ran the developer framework actions
+
+Update rendering of events related to the developer framework. We are now showing the user who ran the action instead of a generic "Developer Framework" actor name
+
+::: {.grid}
+::: {.g-col-6}
+**Before**
+
+<img src="78586ce2-4e9d-4613-9738-b09db49c2d5e.png" alt="Old indicator that says that the developer framework logged an event" style="width: 100%; border: 1px solid #333; box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;">
+
+:::
+
+::: {.g-col-6}
+**After**
+
+<img src="8b643ea9-61c6-445e-bf2a-422d94193ffc.png" alt="New indicator that specifies the user who made updates through the developer framework" style="width: 100%; border: 1px solid #333; box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;">
+:::
+:::
+
+
+<!---Nrichers/sc 2163/revisit once again the getting started page by @nrichers in [#543](https://github.com/validmind/frontend/pull/543)--->
+#### Simplified instructions for developers
+
+We simplified the instructions for getting started with the ValidMind Developer Framework in the Platform UI. These instructions tell you how to use the code snippet for your documentation project with your own model or with one of our code samples.
+
+<img src="bf9b42d3-c74f-41f9-97b5-52911e33eb0d.png" alt="New indicator that specifies the user who made updates through the developer framework" style="width: 80%; border: 1px solid #333; box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;">
+
+## Enhancements
+
+<!---Panchicore/sc 2799/model owner should be able to edit model by @panchicore in [#562](https://github.com/validmind/frontend/pull/562)--->
+Model owner can edit model details page
+: Model owners can now edit the values for fields displayed on the model details page. Previously it was only possible to edit custom fields defined by your organization.
+
+<!---Reducing load time by using localstorage for Auth0 by @even-steven in [#570](https://github.com/validmind/frontend/pull/570)--->
+Reduced load times for the ValidMind Platform UI
+: We reduced the loading times of our Platform UI to improve the user experience.
+
+<!---Added support for filtering model inventory by developers / validators by @even-steven in [#563](https://github.com/validmind/frontend/pull/563)--->
+Added support for filtering model inventory by developers / validators
+: Enhanced the Model Inventory filtering to allow filtering by Developers and Validators.
+
+<!---Support for custom model fields by @panchicore in [#509](https://github.com/validmind/frontend/pull/509)--->
+Support for custom model fields in the model inventory
+: The model inventory has been updated to allow organizations to add custom fields. This enhancement enables administrators to customize the model inventory according to your specific organizational needs.
```

Comment:
```suggestion
: The model inventory has been updated to allow organizations to add custom fields. This enhancement enables administrators to customize the model inventory data schema according to your specific organizational needs. This can be done by accessing **Custom Fields** in the **Settings** page.
```